src/components/common/ColonyActions/CountDownTimer/helpers.ts

```diff
@@ -60,14 +64,14 @@ const useMotionTimeoutPeriods = (colonyAddress = '', motionId: number) => {
   };
 };
 
-export const useMotionCountdown = (
-  state: MotionState,
-  motionId: number,
-  isFullyNayStaked: boolean,
-) => {
+export const useMotionCountdown = (state: MotionState, motionId: number) => {
   const { colony } = useColonyContext();
   const { user } = useAppContext();
   const dispatch = useDispatch();
+  const {
+    motionStakes: { percentage: percentageStaked },
+  } = useStakingWidgetContext();
```

Comment:
How are you accessing Staking Widget context from outside the staking widget?

Reply:
I've moved the context 1 level above in order to get the percentage of the NAY side staked

Reply:
This is the wrong solution for me. The context is staking widget specific, so moving it to encapsulate additional components feels like an ad-hoc refactor (naming wise for sure doesn't make sense). 

You have the motion data available already in the MotionCountdown component. I would prefer it to be passed down to the function, and then if we notice there is a lot of prop drilling going on in the countdown timer components, we can create a separate context for `MotionCountdown`, or extract the motion data into a context that is available for all DefaultMotion subcomponents (which then gets fed to the staking widget context).